pool/app/pool_common/entity.ml

```diff
@@ -274,3 +274,10 @@ module ExperimentType = struct
       PoolError.Field.ExperimentType
   ;;
 end
+
+module Sihl_user = struct
```

Comment:
I have an idea, what if we override this in `service.ml` itself? That way we can ensure that whoever uses `Service.User` uses this version